src/main/java/duke/util/Constants.java

```diff
@@ -0,0 +1,22 @@
+package duke.util;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Constants {
+    public static final String HORIZONTAL_LINE = "____________________________________________________________";
+
+    public static final String GREETINGS = "Hello there, I am Giga Chad Duke\nHow can I help you?";
+
+    public static final String BYE = "Bye, hope to see you again soon!";
+
+    public static final List<String> TASKS = Arrays.asList("list", "bye", "todo", "event",
+            "deadline", "mark", "unmark", "delete", "find");
+
+    public static final String DATA_DIRECTORY = "./data";
+
+    public static final String FILE_NAME = "duke.txt";
+
+    public static final String TEMP_FILE_NAME = "temp.txt";
```

Comment:
good job! Adhering to the naming conventions of constants

Reply:
Agreed with @fredtwt, `Constants` class makes the code clearer and much more readable. 👍